decidim-core/lib/decidim/api/graph_ql_api_transition.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    module GraphQLApiTransition
+      def self.included(base)
+        ActiveSupport::Deprecation.warn(%(
+        GraphQL is in migration mode. Class #{base.name} has been injected with Decidim::Core::GraphQLApiTransition module.
```

Comment:
What's the aim for this? So that developers know that the API is being migrated? But the API still works (both old and new patterns), right?

Reply:
The aim of that Warning is to make sure there are no fields left to be migrated ... and also, a reminder to remove the entire file when everything is finished.

Reply:
Ok!